var/spack/repos/builtin/packages/cprnc/install_rpath.patch

```diff
@@ -0,0 +1,18 @@
+--- a/CMakeLists.txt	2023-12-04 07:01:57.000000000 -0700
++++ b/CMakeLists.txt	2024-11-08 06:53:55.090900241 -0700
+@@ -21,6 +21,7 @@
+ 
+ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+ set(CMAKE_MACOSX_RPATH 1)
++SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
This is the default in Spack, does it not work without this line?

Reply:
Nope, that's what led to the creation of the PR (see PR description). I think the issue was that the library paths in the somewhat unconventional cprnc cmakelists.txt were only added to the build rpath.

Reply:
Oh you mean just this line? Well, maybe. But the other line was definitely needed.

Reply:
yeah, just this line. fine by me to define it twice